src/main/java/com/jadventure/game/Trading.java

```diff
@@ -85,7 +85,7 @@ public void playerSell() {
         Item item = tradeItem(player, npc, itemName);
         if (item != null) {
             if (item != itemRepo.getItem("empty")) {
-                QueueProvider.offer("You have sold a " + item.getName() + " for " + (int)((0.5+0.05*(player.getIntelligence()+player.getLuck()))*item.getProperties().get("value")) + " gold coins.");
+                QueueProvider.offer("You have sold a " + item.getName() + " for " + (int)((0.5+0.02*(player.getIntelligence()+player.getLuck()))*item.getProperties().get("value")) + " gold coins.");
```

Comment:
I think that this should rather get the player's gold before `tradeItem(...)` and the player's gold after `tradeItem(...)` and display the difference. That way if the formula changes, it doesn't require remembering both places it is calculated in.